src/platform/packages/shared/kbn-unified-data-table/src/restorable_state.ts

```diff
@@ -7,14 +7,32 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+import type { InTableSearchRestorableState } from '@kbn/data-grid-in-table-search/src/types';
 import { createRestorableStateProvider } from '@kbn/restorable-state';
+import type { DocumentDiffMode } from './components/compare_documents/types';
 
 type SelectedDocId = string;
 
 export interface UnifiedDataTableRestorableState {
```

Comment:
For the main data grid state, I noticed we're not restoring the grid settings (sample size, density, row heights) unless they've been changed at least once in the tab and written to app state / the URL. I wonder if it's feasible to restore those too?

Reply:
They are a part of app state and not under `@kbn/restorable-state`. Created an issue to address it separately https://github.com/elastic/kibana/issues/226773